hubspot3/crm_associations.py

```diff
@@ -3,10 +3,13 @@
 """
 from hubspot3.base import BaseClient
 from hubspot3.utils import get_log
+from hubspot3.lines import LinesClient
 from enum import Enum
+import itertools
+from typing import Dict, Iterator, List, Union
```

Comment:
```suggestion
import itertools
from enum import Enum
from typing import Dict, Iterator, List, Union

from hubspot3.base import BaseClient
from hubspot3.lines import LinesClient
from hubspot3.utils import get_log
```
Acho que aqui a ordem dos imports seria outra de acordo com o PEP8. Os imports nativos do Python primeiro e os `from` depois dos imports normais em ordem alfabética e depois os do `hubspot3`.

Reply:
Feito!